app/components/login/login.controller.js

```diff
@@ -0,0 +1,28 @@
+;(function () {
+  angular
+    .module("todo")
+    .controller("LoginController", LoginController);
+
+  LoginController.$inject = ["$scope", "$http", "$state", "API", "SessionUser"];
+
+  function LoginController($scope, $http, $state, API, SessionUser) {
+    $scope.signIn = signIn;
+    $scope.isMsgHide = true;
+
+    function signIn() {
+      $http.post(API.BASE + API.AUTH + API.LOGIN, $scope.user)
+        .then(loginSucces, loginError);
+    }
+
+    function loginSucces(response) {
+      SessionUser.getUser();
```

Comment:
`getUser` is async, it returns promise. I think it's better to change state after it finished